scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
Creating the build tools wrappers worked fine, it just used unqualified "gcc" instead of "host-gcc".